src/components/layout/PageLayout.js

```diff
@@ -107,8 +108,21 @@ export function PurePageLayout({ dxData, children, pageContext, ...props }) {
           {/* 
             Set the docsearch index facets defaults
           */}
-          <meta key="docsearch:framework" name="docsearch:framework" content="agnostic" />
-          <meta key="docsearch:version" name="docsearch:version" content="agnostic" />
+          <meta
+            key={GLOBAL_SEARCH_META_KEYS.FRAMEWORK}
+            name={GLOBAL_SEARCH_META_KEYS.FRAMEWORK}
+            content="agnostic"
+          />
+          <meta
+            key={GLOBAL_SEARCH_META_KEYS.VERSION}
+            name={GLOBAL_SEARCH_META_KEYS.VERSION}
+            content="agnostic"
+          />
+          <meta
+            key={GLOBAL_SEARCH_META_KEYS.IMPORTANCE}
+            name={GLOBAL_SEARCH_META_KEYS.IMPORTANCE}
+            content={GLOBAL_SEARCH_IMPORTANCE.RECIPE}
+          />
```

Comment:
This applies to anything that _isn't_ docs. I know right now that's only recipes, but it may not always be that way. I think we should either:

1. If we think we'll always want docs content to rate higher than everything, we should change this constant's name to `GLOBAL_SEARCH_IMPORTANCE.AGNOSTIC`
2. If we don't think that, we should apply this important meta only to recipes
    - Either here: https://github.com/storybookjs/frontpage/blob/main/src/components/screens/IntegrationsCatalog/RecipesDetailScreen/RecipesDetailScreen.js
    - Or by updating the logic (and adding a RecipesLayout component) here: https://github.com/storybookjs/frontpage/blob/main/src/components/layout/PageLayout.js

Reply:
Agreed! Let's make an Agnostic option and we can always tweak from there 😀

Reply:
BOOM! Done